docs/DeveloperGuide.md

```diff
@@ -34,25 +34,25 @@ The ***Architecture Diagram*** given above explains the high-level design of the
 
 [**`Commons`**](#common-classes) represents a collection of classes used by multiple other components.
 
-The rest of the App consists of four components.
+The rest of the App consists of four components:
 
 * [**`UI`**](#ui-component): The UI of the App.
 * [**`Logic`**](#logic-component): The command executor.
 * [**`Model`**](#model-component): Holds the data of the App in memory.
-* [**`Storage`**](#storage-component): Reads data from, and writes data to, the hard disk.
+* [**`Storage`**](#storage-component): Reads and writes data to the hard disk.
 
-Each of the four components,
+Each of the four components
```

Comment:
Should we include a `:` at the end?

Reply:
For this and with the rest, I don't think we should put the colon. Because the transition from sentence to bullet point should be a sentence. I will put commas between bullet points though.